src/api/01-basics/02-multiple-sources.md

```diff
@@ -61,9 +61,9 @@ one to play.
 
 ```html
 <video>
-  <source src="av1.mp4" type="video/mp4; codecs=\"av01.0.04M.08\"">
-  <source src="hevc.mp4" type="video/mp4; codecs=\"hvc1\"">
-  <source src="vp9.mp4" type="video/webm; codecs=\"vp9\"">
+  <source src="av1.mp4" type="video/mp4" codecs="av01.0.04M.08">
```

Comment:
@PaulKinlan I don't believe `codecs` is a valid `<source>` attribute. In this context, `codecs` is a part of the MIME value and should be delimited by `;` from the MIME type.

But you are onto something here. Escaping using `\` is not allowed either, AFAIK. We should probably use single quotes around the `type` attribute value and keep `"` unescaped.

Like this:

```html
<source src="av1.mp4" type='video/mp4; codecs="av01.0.04M.08"'>
<source src="hevc.mp4" type='video/mp4; codecs="hvc1"'>
<source src="vp9.mp4" type='video/webm; codecs="vp9"'>
```